docs/advanced_usage/index.rst

```diff
@@ -61,6 +61,15 @@ Modin provides an implementation of distributed XGBoost machine learning algorit
 :doc:`Modin XGBoost architecture documentation </flow/modin/experimental/xgboost>` for information about implementation and
 internal execution flow.
 
+Experimental APIs
```

Comment:
I'd prefer this section at the top of this page.

Reply:
Moved